ClosedXML/ClosedXML.csproj

```diff
@@ -1,386 +1,53 @@
-    <Compile Include="Excel\PageSetup\XLPrintAreas.cs" />
-    <Compile Include="Excel\Ranges\IXLRanges.cs" />
-    <Compile Include="Excel\Ranges\XLRanges.cs" />
-    <Compile Include="Excel\Rows\IXLRow.cs" />
-    <Compile Include="Excel\PageSetup\XLHeaderFooter.cs" />
-    <Compile Include="Excel\PageSetup\XLHFItem.cs" />
-    <Compile Include="Excel\PageSetup\XLMargins.cs" />
-    <Compile Include="Excel\Rows\IXLRows.cs" />
-    <Compile Include="Excel\Rows\XLRows.cs" />
-    <Compile Include="Excel\Style\IXLAlignment.cs" />
-    <Compile Include="Excel\Style\IXLBorder.cs" />
-    <Compile Include="Excel\Style\IXLFill.cs" />
-    <Compile Include="Excel\Style\IXLNumberFormat.cs" />
-    <Compile Include="Excel\Style\XLAlignment.cs" />
-    <Compile Include="Excel\Style\XLBorder.cs" />
-    <Compile Include="Excel\Style\XLFill.cs" />
-    <Compile Include="Excel\Style\XLNumberFormat.cs" />
-    <Compile Include="Excel\Columns\XLColumnParameters.cs" />
-    <Compile Include="Excel\PageSetup\XLPageSetup.cs" />
-    <Compile Include="Excel\Ranges\XLRangeParameters.cs" />
-    <Compile Include="Excel\Rows\XLRowParameters.cs" />
-    <Compile Include="Excel\XLWorkbook_Save.cs" />
-    <Compile Include="Excel\Coordinates\IXLAddress.cs" />
-    <Compile Include="Excel\Cells\IXLCell.cs" />
-    <Compile Include="Excel\Columns\IXLColumn.cs" />
-    <Compile Include="Excel\Ranges\IXLRange.cs" />
-    <Compile Include="Excel\IXLWorksheet.cs" />
-    <Compile Include="Excel\IXLWorksheets.cs" />
-    <Compile Include="Excel\Style\IXLFont.cs" />
-    <Compile Include="Excel\Style\IXLStyle.cs" />
-    <Compile Include="Excel\Style\IXLStylized.cs" />
-    <Compile Include="Excel\Style\XLFont.cs" />
-    <Compile Include="Excel\Style\XLStyle.cs" />
-    <Compile Include="Excel\Coordinates\XLAddress.cs" />
-    <Compile Include="Excel\Cells\XLCell.cs" />
-    <Compile Include="Excel\Columns\XLColumn.cs" />
-    <Compile Include="Excel\Ranges\XLRange.cs" />
-    <Compile Include="Excel\Rows\XLRow.cs" />
-    <Compile Include="Excel\XLWorkbook.cs" />
-    <Compile Include="Excel\XLWorksheet.cs" />
-    <Compile Include="Excel\XLWorksheetInternals.cs" />
-    <Compile Include="Excel\XLWorkbookProperties.cs" />
-    <Compile Include="Excel\XLWorksheets.cs" />
-    <Compile Include="Excel\XLWSContentManager.cs" />
-    <Compile Include="Extensions.cs" />
-    <Compile Include="PathHelper.cs" />
-    <Compile Include="Properties\AssemblyInfo.cs" />
-    <Compile Include="Properties\AssemblyVersionInfo.cs" />
-    <Compile Include="Excel\Drawings\Style\XLDrawingProperties.cs" />
+    <None Include="..\.editorconfig" Link=".editorconfig" />
   </ItemGroup>
+
   <ItemGroup>
-    <None Include="..\.editorconfig">
-      <Link>.editorconfig</Link>
-    </None>
-    <None Include="ClosedXML.snk" />
-    <None Include="packages.config">
-      <SubType>Designer</SubType>
-    </None>
+    <PackageReference Include="DocumentFormat.OpenXml" Version="2.7.2" />
```

Comment:
Did you consider upgrading to 2.8.1? (https://github.com/OfficeDev/Open-XML-SDK/tree/v2.8.0#change-log)

Reply:
I try to keep to the lowest common denominator. Users are free to upgrade to a higher version of OpenXML. Is there any functionality in OpenXML 2.8+ that we'd want to use that isn't in 2.7.2?

Reply:
I thought about bug fixes that possibly could affect us, but you are right, it's better to stick to the lowest suitable version.